AdobeStockClient/Model/SearchParametersProvider/Sorting.php

```diff
@@ -29,7 +29,7 @@ public function apply(SearchCriteriaInterface $searchCriteria, SearchParameters
     {
         $resultsOrders = Constants::getSearchParamsOrders();
         foreach ($searchCriteria->getSortOrders() as $sortOrder) {
-            if (false !== ($sortOrderField = array_search($sortOrder->getField(), $resultsOrders))) {
+            if (false !== ($sortOrderField = array_search($sortOrder->getField(), $resultsOrders, null))) {
```

Comment:
Why do you think we need to pass `null` explicitly?

Reply:
Because null is the default value, it does not change anything but removes the warning:
> Third parameter should be provided to clarify if types safety is important in this context.